experiments/kan_benchmark/environment.yml

```diff
@@ -0,0 +1,10 @@
+name: kan_benchmark
+channels:
+  - conda-forge
+dependencies:
+  - numpy<1.24
```

Comment:
do we know why we are using this restriction on `numpy`? is it related to the model?

Reply:
I actually just copied this from the .yml file from the n-hits experiment. I can remove it!

Reply:
Removed restriction on numpy.